client/lib/api/UserPlant.dart

```diff
@@ -92,4 +92,17 @@ plantId:\t\t$plantId
 ''';
   }
 
+  Plant toPlant() {
+    return new Plant(
+      id: id,
+      name: nickname,
+      imageName: imageName,
+      waterScale: 2,
+      sunScale: 2,
+      waterText: 'Plant needs water.',
+      sunText: 'Plant needs sun.',
+      description: 'This is a plant.'
+    );
+  }
+
```

Comment:
Why is this in the UserPlants class and not in the Plant class. These two have a common factor but are not the same